ERCS/erc-social_sbt.md

```diff
@@ -0,0 +1,124 @@
+---
+eip: draft
+title: Social SBT
+description: Social SBT, A Soul-Bound Token Standard for DAO Governance
+author: Ali Bertay SOLAK (@alibertay)
+status: Draft
```

Comment:
```suggestion
discussions-to:
status: Draft
```
Please create a discussions topic in Eth Magicians with a link to this PR.  https://ethereum-magicians.org/c/ercs/57